docs/compose-file.md

```diff
@@ -92,28 +42,29 @@ definition.
 
 Configuration options that are applied at build time.
 
-In version 1 this must be given as a string representing the context.
+In the [version 1 file format](#version-1), `build` must be specified as a
+string containing the path to the build context.
```

Comment:
Should we maybe start with v2, and move the v1 format to the bottom of each section?